.github/workflows/visual-diff.yml

```
@@ -0,0 +1,145 @@
+name: Visual diff
+
+concurrency:
+  group: visual-diff-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    types:
+      - opened
+      synchronize
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)